src/components/scripture/books.ts

```diff
@@ -1206,7 +1206,7 @@ const books = [
     chapters: [14],
   },
   {
-    names: 'Jude Jude'.split(' '),
+    names: 'Jude Jud Jude'.split(' '),
```

Comment:
Not sure why Jude was listed twice here but the second one is not needed. `Jud` conflicts with _Judges_.

Reply:
Was just thinking that. Jud should be judges. My bad.